glossary.yml

```diff
@@ -832,6 +838,11 @@
     def: >
       The folder or directory location that the program is operating in. Any action
       taken by the program occurs relative to this directory.
+  pt:
+    term: "diretório de trabalho"
```

Comment:
I don't know why these diffs are showing up - they're in `master` and in `governance`, so they...shouldn't?